src/gosling-track/gosling-track.ts

```diff
@@ -84,6 +85,8 @@ function GoslingTrack(HGC: import('@higlass/types').HGC, ...args: any[]): any {
         private mRangeBrush: LinearBrushModel;
         private _xScale!: ScaleLinear<number, number>;
         private _yScale!: ScaleLinear<number, number>;
+        private position!: [number, number]; // left and top offsets of a track
+        private dimensions!: [number, number]; // width and height of a track
```

Comment:
Changing these to JSDoc comments means the information is accessible when hovering the property anywhere in the file:

```suggestion
        /** left and top offsets of a track */
        private position!: [number, number];
        /** width and height of a track */
        private dimensions!: [number, number];
```

![image](https://user-images.githubusercontent.com/24403730/180234457-80021e14-be91-4d4a-bcef-be387e1cc170.png)